src/server.cpp

```diff
@@ -742,7 +747,7 @@ void Server::AsyncRunStep(bool initial_step)
 			for (std::unordered_map<std::string, ModMetadata *>::const_iterator
 				it = m_mod_storages.begin(); it != m_mod_storages.end(); ++it) {
 				if (it->second->isModified()) {
-					it->second->save(getModStoragePath());
+					//it->second->save(getModStoragePath());
```

Comment:
Can't you drop the entire loop then?

Reply:
I haven't gotten rid of all the unneeded code yet.